CalibTracker/SiStripESProducers/plugins/real/SiStripApvSimulationParametersESProducer.cc

```diff
@@ -47,34 +47,40 @@ SiStripApvSimulationParametersESSource::SiStripApvSimulationParametersESSource(c
   for (const auto x : conf.getParameter<std::vector<double>>("apvBaselines_zBinEdges")) {
     zBinEdges_.push_back(x);
   }
-  for (const auto x : conf.getParameter<std::vector<double>>("apvBaselines_rBinEdges_TID")) {
+  for (const auto x : conf.getUntrackedParameter<std::vector<double>>("apvBaselines_rBinEdges_TID")) {
```

Comment:
These parameters (and below) affect physics output and thus should remain tracked, no?

Reply:
no. They don't affect physics. The producer is used only to produce a payload at the moment which is read by the GloblTag `ESSource`.